features/proxy_config.feature

```diff
@@ -61,3 +61,43 @@ Feature: Proxy configuration
            | xenial  |
            | bionic  |
            | focal   |
+
+    @series.lts
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attach command when proxy is configured
```

Comment:
We combined the apt proxy tests into the existing scenario to avoid spinning up a proxy instance twice. We may want to do that here again (combine this into the previous scenario), or maybe the previous scenario is getting too long and this should stay separate. thoughts?

Reply:
Oops I meant to cancel this comment. I noticed that this scenario requires a vm for livepatch